presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4788,6 +4788,7 @@ public void testShowSchemasLike()
         assertEquals(result.getOnlyColumnAsSet(), ImmutableSet.of(getSession().getSchema().get()));
     }
 
+    @Test
```

Comment:
this PR already depends on 3 other PRs, I do not want to add another dependency for such minor change